src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,42 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
assertj 에서 제공하는 컬렉션용 검증 api를 활용해 보셔도 좋을 것 같아요 
```suggestion
        assertThat(numbers).hasSize(3);
```